plugins/pencilblue/controllers/admin/sites/manage_sites.js

```diff
@@ -14,64 +14,134 @@
  You should have received a copy of the GNU General Public License
  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-
-
+const Promise = require('bluebird');
 module.exports = function(pb) {
 
-    var util = pb.util;
+    const SUB_NAV_KEY = 'sites_manage';
+    const SITE_COLLECTION = 'site';
+    const PER_PAGE = 2;  // TODO: Increase to 50 before release
 
     /**
      * Renders a view to display and manage all the sites
      * @constructor
      * @extends BaseController
      */
-    function ManageSites(){}
-    util.inherits(ManageSites, pb.BaseController);
+    class ManageSites extends pb.BaseController {
+
+        constructor() {
+            super();
+            this.dao = Promise.promisifyAll(new pb.DAO());
+        }
+        /**
+         * Render view to manage sites.
+         * @method render
+         * @param {Function} cb - callback function
+         */
+        render(cb) {
+            this._getSites()
+                .then(([activeSites, inactiveSites, activeCount, inactiveCount]) => {
+                    let angularObjects = pb.ClientJs.getAngularObjects({
+                        navigation: pb.AdminNavigation.get(this.session, ['site_entity'], this.ls, this.site),
+                        pills: pb.AdminSubnavService.get(SUB_NAV_KEY, this.ls, SUB_NAV_KEY),
+                        activeSites,
+                        inactiveSites,
+                        activeCount,
```

Comment:
pass along page count so we don't need to update PER_PAGE on the front end as well- don't forget to rip it out of the angular file ;)